apps/api/src/auth/keycloak.ts

```diff
@@ -1,21 +1,36 @@
+import { Logger } from '@nestjs/common'
 import { RealmViewSupporters, ViewSupporters } from '@podkrepi-bg/podkrepi-types'
 
 export function isAdmin(user: KeycloakTokenParsed): boolean {
```

Comment:
This function becomes quite complex, maybe we need to add a unit test to confirm all the expected cases